react-native/android/src/main/jni/Application.mk

```diff
@@ -12,6 +12,7 @@ APP_CPPFLAGS := -std=c++14
 APP_CPPFLAGS += -frtti
 APP_CPPFLAGS += -fexceptions
 APP_CPPFLAGS += -DREALM_HAVE_CONFIG
+APP_CPPFLAGS += -fomit-frame-pointer
```

Comment:
Was getting compile errors in the assembly in this file https://github.com/realm/realm-js/blob/master/src/android/jsc_override.cpp before this change - I assume it is due to using a newer version of the ndk.